scalyr_agent/line_matcher.py

```diff
@@ -82,34 +116,44 @@ def __init__( self , max_line_length=5*1024, line_completion_wait_time=5*60 ):
         self.__line_completion_wait_time = line_completion_wait_time
         self.__partial_line_time = None
 
-    def readline( self, file_like, current_time ):
+    def readline( self, file_like, current_time, current_file=None ):
+        """
+        Reads a line from a `file_like` object
```

Comment:
Document `current_file`.  I know the other arguments aren't documented, but this one is not intuitive to me so worth documenting now IMO.